### PR TITLE
modify Design doc ttl-after-finish upstream

### PR DESCRIPTION
**Pull request by chienfuchen32**

### CODE REVIEW

**Comment by ydFu** [comment]
@chienfuchen32 
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Review by Aut0R3V — APPROVED** [review]

File: content/en/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 1]
```diff
@@ -78,5 +78,5 @@ very small. Please be aware of this risk when setting a non-zero TTL.
 
 * [Clean up Jobs automatically](/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically)
 
-* [Design doc](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/0026-ttl-after-finish.md)
+* [Design doc](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/592-ttl-after-finish/README.md)
```
Comment (bradtopol) [inline content/en/docs/concepts/workloads/controllers/ttlafterfinished.md:81]: verified that new link fixes the previous broken link and points to the correct KEP! Great job!

**Comment by bradtopol** [comment]
Verified that new link fixes the previous broken link and points to the correct KEP! Great job!
/lgtm
/approve

**Pull request merged**